composer.json

```diff
@@ -35,7 +36,7 @@
             "sed -i.bak 's/<!--<severity>0<\\/severity>-->/<severity>0<\\/severity>/' CakePHP/ruleset.xml",
             "rm -f CakePHP/ruleset.xml.bak"
         ],
-        "add-standard" : "phpcs --config-set installed_paths $(pwd)",
+        "add-standard" : "phpcs --config-set installed_paths $(pwd),$(pwd)/vendor/slevomat/coding-standard",
```

Comment:
How does this new dependency change the install/setup process for end users?

Reply:
They'll have to do the same thing that's done here. Add the path to slevomat standard too to `installed_paths`.

Reply:
That's annoying. Would we be able to use composer hooks to automatically do that?

Reply:
It might be possible to do that automatically with composer hooks, but I am not too sure we should be implicitly overriding phpcs config.

For e.g. if someone already has phpcs installed globally and customized `installed_paths`, installing our codesniffer shouldn't overwrite `installed_paths`.